### DISCUSSION TITLE
Documentation incorrect for nixos flake?

### DISCUSSION THREAD

**Discussion by DrTeagle** [outfoxxed/hy3#134] [Q&A] [answered]
Howdy, 
I dont want to open an issue as I think this might just be an error in my understanding, but in the documentation for hyprland, to follow the git version one puts 
```nix
    inputs.hyprland.url = "git+https://github.com/hyprwm/hyprland?submodules=1";
```
but here it says to put 
```nix
    inputs.hyprland.url = "github:hyprwm/Hyprland";
```
whenever I do the later my build fails, whenever I do the former my plugin build fails saying that the version is mismatching

**Comment by outfoxxed** [comment, marked as answer]
mb, forgot to update that bit of the readme, you want ?submodules=1

**Comment by DrTeagle** [comment]
ah, so I wasnt trippin

**Comment by DrTeagle** [comment]
in which case how can I troubleshoot the plugin not launching? I already have 
```nix
inputs.hy3.inputs.hyprland.follows = "hyprland"
```

**Comment by DrTeagle** [comment]
I am dumb I hadnt updated the system version of hyprland

**Comment by outfoxxed** [comment]
fixed docs
